packages/flutter_tools/lib/src/commands/run.dart

```diff
@@ -68,12 +69,22 @@ class RunCommand extends RunCommandBase {
         defaultsTo: false,
         negatable: false,
         help: 'Start in a paused mode and wait for a debugger to connect.');
+    argParser.addFlag('pub',
```

Comment:
People who use this command from inside a continuous integration system will probably want to explicitly skip this step to prevent us from touching the network.